AlembicImporter/Packages/com.unity.formats.alembic/Documentation/com.unity.formats.alembic.md

```diff
@@ -6,7 +6,7 @@ The Alembic package supports the import and playback of Meshes, Points and Camer
 
 ## Requirements
 
-The Alembic Package is compatible with Unity Editor 2018.2 and above.
+The Alembic Package is compatible with Unity Editor 2018.1 and above.
```

Comment:
are we going to test on 2018.1 in the end?

Reply:
@Alexis already did a pass, no problem were found